packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1546,11 +1546,13 @@ namespace Tpetra {
     /// does their order change, as a result of calling this method.
     ///
     /// \param newColMap [in] New column Map.  Must be nonnull.
-    ///   The column map's list of global indices on each process must be in a certain order:
+    ///   Within Tpetra, there are no particular restrictions on the column map.
+    ///   However, if this graph will be used in Xpetra, Ifpack2, or MueLu,
+    ///   the column map's list of global indices on each process must be in a certain order:
     ///   locally owned GIDs (sorted), followed by remote GIDs (in order of
```

Comment:
Locally owned GIDs don't have to be sorted; they just need to be in the same order as in the domain Map.  The domain Map defines the "locally owned GIDs" for each process.